package.json

```diff
@@ -6,36 +6,26 @@
     "react-scripts": "1.1.0"
   },
   "dependencies": {
-    "aphrodite": "^1.2.5",
-    "axios": "^0.17.1",
-    "bcrypt": "^2.0.1",
-    "body-parser": "^1.18.2",
-    "draft-js": "^0.10.5",
-    "draftjs-to-html": "^0.8.4",
-    "express": "^4.16.2",
-    "foreman": "^2.0.0",
-    "glyphicons": "^0.2.0",
-    "html-to-draftjs": "^1.4.0",
-    "marked": "^0.3.12",
-    "moment": "^2.22.2",
-    "mongoose": "^5.0.4",
-    "nodemon": "^1.14.12",
-    "prop-types": "^15.6.0",
-    "react": "^16.2.0",
-    "react-datepicker": "^1.5.0",
-    "react-dom": "^16.2.0",
-    "react-draft-wysiwyg": "^1.12.13",
-    "react-router-dom": "^4.2.2"
+    "foreman": "^3.0.1"
   },
   "peerDependencies": {
     "draft-js": "^0.10.0",
     "immutable": "^4.0.0"
   },
+  "description": "ReactCRUD is a basic full Javascript CRUD made with love by three french students and for the development of there main application Sooonr.",
   "scripts": {
-    "start": "react-scripts start",
     "start-dev": "nf start -p 3000",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
+    "install-all": "npm install; cd server && npm install; cd ../client && npm install"
```

Comment:
The command doesn't work for me with semicolons. I had to replace them with the operator "&&" :
`"install-all": "npm install && cd server && npm install && cd ../client && npm install"`